.pre-commit-config.yaml

```diff
@@ -1,28 +1,28 @@
 default_language_version:
   node: 18.7.0
-  python: python3.10
+  python: python3.11
 repos:
   - repo: https://github.com/psf/black
-    rev: 22.10.0
+    rev: 23.1.0
```

Comment:
This and all below – looks like we’ll need to do the same upgrades to versions defined in Poetry?